fastlane/lib/fastlane/actions/device_grid/device_grid.rb

```diff
@@ -0,0 +1,104 @@
+# rubocop:disable Style/IndentationConsistency
+# We disable this rule since we intend opening HTML labels
+module Danger
+  class Dangerfile
+    module DSL
+      # A danger plugin: https://github.com/danger/danger
+      class DeviceGrid < Plugin
+        # @param languages: Array of languages you want to see (e.g. [en, de])
+        # @param devices: Array of deviecs you want to see (e.g. ["iphone4s", "ipadair"])
+        # @param prefix_command: Prefix the `fastlane run appetize_viewing_url_generator` command with something
+        #   this can be used to use `bundle exec`
+        def run(languages: nil, devices: nil, prefix_command: nil)
+          public_key = nil
+          [".fastlane/appetize_public_key.txt", "fastlane/appetize_public_key.txt"].each do |current| # not using Dir[...] since it doesn't show hidden files
```

Comment:
Great idea, however that's a Dangerfile plugin. Or do you mean a command line interface, something like `fastlane fastlane_dir`?